docs/admin/disruptions.md

```diff
@@ -34,11 +34,13 @@ disruption budget to be violated.
 
 ## Specifying a PodDisruptionBudget
 
-A `PodDisruptionBudget` has two components: a selector to specify the set of
-pods, and a description of the minimum number of available pods for a disruption
-to be allowed.  The latter can be either an absolute number or a percentage.  In
-typical usage, a single budget would be used for a collection of pods managed by
-a controller—for example, the pods in a single ReplicaSet.
+Currently, a `PodDisruptionBudget` has two components: a selector to specify the
+set of pods, and a description of the minimum number of available pods for a
```

Comment:
I don't think this first part is quite clear. I would say "a description of the minimum number of pods that must be available after an eviction for a voluntary eviction to be allowed" instead of "a description of the minimum number of available pods for a disruption to be allowed"

Also, I would add the 100% example you added to the proto comment, since it makes it more concerete ("So for example you can prevent all voluntary evictions by specifying "100%".)